versioned_docs/version-8.8/components/connectors/protocol/rest.md

```diff
@@ -83,12 +83,32 @@ As an alternative to using JVM properties, the proxy settings can also be set wi
 The HTTPS properties also use the `http.nonProxyHosts` or `CONNECTOR_HTTP_NON_PROXY_HOSTS` property to specify non-proxy hosts.
 :::
 
-| Proxy config set | nonProxyHost config set | valid login provided      | domain1.com (proxied site, no auth required) | domain2.com (proxied site, auth required) | domain3.com (nonProxyHost site) |
-| ---------------- | ----------------------- | ------------------------- | -------------------------------------------- | ----------------------------------------- | ------------------------------- |
-| ❌               | N/A                     | N/A                       | no proxy                                     | no proxy                                  | no proxy                        |
-| ✅               | ❌                      | ✅                        | proxy                                        | proxy                                     | proxy                           |
-| ✅               | ✅                      | ❌ (incorrect or missing) | proxy                                        | Auth error                                | no proxy                        |
-| ✅               | ✅                      | ✅                        | proxy                                        | proxy                                     | no proxy                        |
+#### Learn how the proxy configuration works
+
+The process consists of two main steps: configuration and request handling.
+
+##### Set your configuration
+
+First, define how the proxy should behave.
+These are the available configuration options:
```

Comment:
It’s a bit unclear whether these are sequential setup steps or simply available configuration options.

If they’re steps to follow, consider rephrasing to something like:
“Take the following steps to configure proxy behavior:”

If they’re independent/optional settings, consider using an unordered list instead of a numbered one for clarity.